.nvmrc

```diff
@@ -1 +1 @@
-v14.17.3
```

Comment:
Needed to bump to the latest version of node 14 to support the current version of `execa`

Reply:
The workflow "Integration tests: CLI" is failing with the error below

```
To run storybook you need to have node 16 or higher
```

https://github.com/oblador/loki/actions/runs/4655936027/jobs/8239071634?pr=450
![image](https://user-images.githubusercontent.com/3720424/232257851-6fa5bcda-3189-415c-a473-c89a8868a055.png)

Reply:
I guess we need to use Node.js 16.x (or newer) here 👇 
https://github.com/oblador/loki/blob/7ca6aa0101151dd7ec56652708cdafd1083079cf/.github/workflows/tests.yml#L53

Reply:
According to their official release schedule, Node.js 14.x will reach end-of-life this month

https://github.com/nodejs/release#release-schedule
![image](https://user-images.githubusercontent.com/3720424/232257978-e1a3ef93-d00f-4d30-ad4f-cd5630f286bb.png)

Reply:
Should we go ahead and remove support for Node.js 14.x (and older)? @oblador

Reply:
Let's!

Reply:
#451